scipy/constants/__init__.py

```diff
@@ -5,316 +5,104 @@
-
-======================================================================  ====
-%(constant_names)s
-======================================================================  ====
-
-
-Units
-=====
-
-SI prefixes
------------
-
-============  =================================================================
-``yotta``     :math:`10^{24}`
-``zetta``     :math:`10^{21}`
-``exa``       :math:`10^{18}`
-``peta``      :math:`10^{15}`
-``tera``      :math:`10^{12}`
-``giga``      :math:`10^{9}`
-``mega``      :math:`10^{6}`
-``kilo``      :math:`10^{3}`
-``hecto``     :math:`10^{2}`
-``deka``      :math:`10^{1}`
-``deci``      :math:`10^{-1}`
-``centi``     :math:`10^{-2}`
-``milli``     :math:`10^{-3}`
-``micro``     :math:`10^{-6}`
-``nano``      :math:`10^{-9}`
-``pico``      :math:`10^{-12}`
-``femto``     :math:`10^{-15}`
-``atto``      :math:`10^{-18}`
-``zepto``     :math:`10^{-21}`
-============  =================================================================
-
-Binary prefixes
----------------
+- The second :math:`\mathrm{s}` as the SI unit of time is derived from the
+  unperturbed ground-state hyperfine transition frequency of the caesium-133
+  atom :math:`\Delta \nu_{\mathrm{Cs}}`.
 
-============  =================================================================
-``kibi``      :math:`2^{10}`
-``mebi``      :math:`2^{20}`
-``gibi``      :math:`2^{30}`
-``tebi``      :math:`2^{40}`
-``pebi``      :math:`2^{50}`
-``exbi``      :math:`2^{60}`
-``zebi``      :math:`2^{70}`
-``yobi``      :math:`2^{80}`
-============  =================================================================
+- The metre :math:`\mathrm{m}` as the SI unit of length is derived from the
+  speed of light in vaccum :math:`c`.
 
-Mass
-----
+- The kilogram :math:`\mathrm{kg}` as the SI unit of mass is derived from the
+  Planck constant :math:`h`.
 
-=================  ============================================================
-``gram``           :math:`10^{-3}` kg
-``metric_ton``     :math:`10^{3}` kg
-``grain``          one grain in kg
-``lb``             one pound (avoirdupous) in kg
-``pound``          one pound (avoirdupous) in kg
-``blob``           one inch version of a slug in kg (added in 1.0.0)
-``slinch``         one inch version of a slug in kg (added in 1.0.0)
-``slug``           one slug in kg (added in 1.0.0)
-``oz``             one ounce in kg
-``ounce``          one ounce in kg
-``stone``          one stone in kg
-``grain``          one grain in kg
-``long_ton``       one long ton in kg
-``short_ton``      one short ton in kg
-``troy_ounce``     one Troy ounce in kg
-``troy_pound``     one Troy pound in kg
-``carat``          one carat in kg
-``m_u``            atomic mass constant (in kg)
-``u``              atomic mass constant (in kg)
-``atomic_mass``    atomic mass constant (in kg)
-=================  ============================================================
+- The ampere :math:`\mathrm{A}` as the SI unit of electric current is derived
```

Comment:
This would read better as `ampere (A)` rather than `ampere A`. And same for next lines.